test/parallel/test-fs-watchfile.js

```diff
@@ -19,7 +18,7 @@ assert.throws(function() {
   fs.watchFile(new Object(), function() {});
 }, /Path must be a string/);
 
-const enoentFile = path.join(fixtures, 'non-existent-file');
+const enoentFile = path.join(common.tmpDir, 'non-existent-file');
```

Comment:
`refreshTmpDir()` should be called before creating files in `common.tmpDir` but there's no reason why the path to the file can't be created and saved first as it is here.